docs/03.reference/01.functions/directorycreate/_examples.md

```diff
@@ -1 +1,4 @@
-*There are currently no examples for this function.*
+```cfc
```

Comment:
Hey @andyj, thanks for this. Could you change the language annotation to be `luceescript` rather than `cfc`?. I think this is preventing the build from running successfully.

D